tech-team-practices.md

```diff
@@ -0,0 +1,40 @@
+
+## Our goals
+
+Here are the goals we optimize for in organizing our team work practices.
+
+- Give team members insight into what one another is working on
+- Provide some social accountability to get stuff done
+- Look for opportunities to work together and/or support one another's work
+- Look for opportunities to help one another get stuff done faster or more effectively
+- Do this with high efficiency so that people can quickly get on the same page and focus efforts around decision-making and collaboration
+- Do all of this in a 💯 remote-friendly way since we are split across many different time-zones.
+
+## Defining our active projects and to-do items
+
+**Active projects list**: The 2i2c handbook has [a projects page](https://2i2c.org/handbook/projects/) that has details about major projects that we run at 2i2c. We use this as {term}`Source of Truth` for the major projects that we are working on, and this can be a combination of development-focused things and higher-level project-focused things. These projects should contain links to the location that daily "project management" and goal-setting are happening.
+
+**Per-project to-dos and conversation**: Generally speaking specific projects will have their own place for team conversation, to-do items, and strategy. This is generally *not* in the 2i2c handbook, but there should always be links and references in the handbook that points somebody in the right direction.
+
+**Discussion around team practices**: For discussion around strategy that doesn't have a natural place elsewhere, we use `handbook` issues and PRs. If an issue, it should be actionable (and "closeable"), generally in a way that results in a change to the handbook. If a PR, it should be a proposal to change the handbook and thus update some strategic information in there.
+
+## Bi-weekly team syncs
+
+Every two weeks we synchronize our team goals and practices. Here's how to conduct a team sync.
+
+1. **Create an issue in `handbook/`**. This is where we'll keep track of the "to-do" for the team sync.
+2. **Clean up [the HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw?both) for the sync**. We'll use [this HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw?both) to run the team sync. At the top is a set of questions to answer, and we can We'll initially use the HackMD to give responses. You can create it from the following template:
+3. **Team members give their responses**. You can copy/paste the template, and then give your responses below. You shouldn't feel forced to add content if you can't think of anything, use it as much as is useful. Just remember that others depend on this information to know what one another is up to!
+4. **We'll leave the HackMD open for 48hrs**. That should give enough time for those of us across many time zones to response.
```

Comment:
```suggestion
4. **We'll leave the HackMD open for 48hrs**, not counting weekends or holidays. That should give enough time for those of us across many time zones to respond.
```

Explicitly say that the 48h doesn't count holidays & weekends. 'Holidays' across countries is complicated but we can probably figure that out if it becomes a problem.